STM32F303K6Tx/Core/Src/main.c

```diff
@@ -132,7 +137,47 @@ void HAL_ADC_ConvCpltCallback(ADC_HandleTypeDef* hadc) {
   }
 }
 
-void HAL_CAN_RxFifo0MsgPendingCallback(CAN_HandleTypeDef* hcan) {
+/*fields[0] = PRESSURE_HIGH;
+fields[1] = PRESSURE_LOW_1;
+fields[2] = PRESSURE_LOW_2;*/
+
+void psyduck_entry() {
```

Comment:
Can you move psyduck_entry to a seperate file called psyduck_entry.c or something.

Reply:
And just add a header file psyduck_entry.h and call the function in main. The point is so that its less messy in here

Reply:
Also the way psyduck_entry should go is:

```
void psyduck_entry() {
   // Do setup things

  while (1) {
    // Do run things
  }
}
```